pipenv/project.py

```diff
@@ -67,13 +67,11 @@ def _build_package_list(self, package_section):
                     else:
                         ps.update({k: v})
                 else:
-                    if not (is_installable_file(k) or is_installable_file(v) or
-                            any(file_prefix in v for file_prefix in ['path', 'file'])):
-                        ps.update({k: v})
+                    ps.update({k: v})
             else:
                 if not (any(is_vcs(i) for i in [k, v]) or
-                        any(is_installable_file(i) for i in [k, v]) or
-                        any(is_valid_url(i) for i in [k, v])):
```

Comment:
Any reasons we're deviating from the use of `i` in the other checks? They should probably all be `entry` or `i`.

Reply:
Yeah I was contemplating swapping everything to `entry` but I had to re-think all my logic anyway so I just left it as `i` across the board (which is not very descriptive but uh, sometimes it keeps our lines under 80 characters?)